pyproject.toml

```diff
@@ -8,3 +8,11 @@ exclude = '''
   | \.git
 )/
 '''
+
+[tool.ruff]
+line-length = 79
+
+[tool.ruff.lint.isort]
+lines-after-imports = 2
+force-sort-within-sections = false
```

Comment:
@findtopher did you mean to make this `true`? Since you resolved my comment without disputing it.
[ignore direct vs from imports](https://github.com/voxel51/fiftyone/pull/4551/commits/b1a41ba300832130938e42edde9557168085b036)